src/geosuggest.css

```diff
@@ -70,3 +70,11 @@
 .geosuggest__item__matched-text {
   font-weight: bold;
 }
+.geosuggest__item__footer {
```

Comment:
Instead of all this styling (lines 73–80) which is rather distractive I would just add a top border to the footer item:

```css
.geosuggest__item__footer {
  border-top: 1px solid #ddd;
}
```